R/param_class_weights.R

```diff
@@ -0,0 +1,22 @@
+#' Parameters for class weights for imbalanced problems
+#'
+#' This parameter can be used to moderate how much influence certain classes
+#' receive during training.
+#'
+#' @inheritParams Laplace
+#' @details
+#' Used in `lantern::lanter_logistic_reg()` and `lantern::lanter_mlp()`
+#' @examples
+#' class_weights()
+#' @export
+class_weights <- function(range = c(1, 10), trans = NULL) {
+  new_quant_param(
+    type = "double",
+    range = range,
+    inclusive = c(TRUE, TRUE),
+    trans = trans,
+    default = 1,
+    label = c(class_weights = "Minority Class Weight"),
```

Comment:
If the parameter is the weight for a single class, do we want the name to reflect that?

Reply:
The model parameters will be more general (e.g. a weight vector) but will accept a single number that will apply a weight to minority class. I'd like to leave it as-is so that the dials parameter matches the argument name.